setup_bibliography.Rmd

```diff
@@ -0,0 +1,58 @@
+bibliography: library.bib
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## Bibliography Managers
+Bibliography managers are important to keep all the literature you work with well organized.
+This page will help you understand pros and cons of different bibliography managers as well as help you set up Zotero, a free and open-source bibliography manager.
+
+## Comparison
+There are many different bibliography managers available, e.g. Citavi, Colwiz, EndNote, JabRef, Mendeley, Zotero and ZoteroBib.
+Since we focus on open-source software, software like Citavi and EndNote are out of question.
+Of the named software, only JabRef, Zotero and ZoteroBib are free and open source.
+For a detailed feature comparison, you may want to look [here (german)](https://ub.fau.de/schreiben-publizieren/literaturverwaltung/) or [here (english)](https://en.wikipedia.org/wiki/Comparison_of_reference_management_software).
+While the following paragraphs will cover Zotero, please feel free to use any of the other open source bibliography managers. As long as they can handle the [BibTex format](https://de.wikipedia.org/wiki/BibTeX) it should be all fine.
+
+## Zotero setup
+To get started with Zotero, there are some components that should be set up:
+
+### Zotero itself
+Windows: 
+Best way is to install via chocolatey package manager. Run in command line: `choco install zotero`
+
+Linux:
+use the included package manager, there is very likely it is already there.
+
+Alternative: download and install via the [Zotero website](https://www.zotero.org/)
+
+
+### BetterBibTex Addon for Zotero [installation instructions](https://retorque.re/zotero-better-bibtex/installation/)
+Prerequisite: Zotero needs to be set up before.
+This enables an automatic export of the bib file, among several other features.
+    
+### Zotero-picker Addon for Atom. [installation instructions](https://atom.io/packages/zotero-picker)
+Prerequisites: Zotero, BetterBibTex Addon
+It enables you to quote and cite directly in Atom.
+    
+### (optional, recommended) Zotero connector for your browser [installation instructions](https://www.zotero.org/download/connectors)
+Prerequisite: Zotero    
+This enables you to add the websites or PDFs you view in your browser directly to your bibliography. And as much as possible will be automatically tagged.
+    
+
+## Using Zotero
+This should be pretty straightforward. To learn about the available functions, have a look at the [Zotero Quick Start Guide](https://www.zotero.org/support/quick_start_guide)
+
+## Finalizing a document
+Navigate with command prompt / bash to the folder your .md document (and in this example, also the .bib file) is stored. The following command will generate a new file in HTML format:
+pandoc .\homework.md --bibliography=mybib.bib -o .\Outputs\output.html
+- `-o` defines the output location to be a subfolder of the current location.
+- The default format is HTML, so this is what will be generated.
```

Comment:
there is one more step: you have to actually export the bibliography (or parts of it) to `.bib` (for Zotero at least).
This can be done from the file menu of Zotero.
Helpfully, there is also an option to automatically update the `.bib`, whenever the originating library changes (though this will of course require a new git commit).
Would be nice to add this, or (preferably?) link to some documentation to this that probably already exists somewhere on the interwebs.

Reply:
Using BetterBibTex, this manual process is a thing of the past. It will handle the export automatically :)
I should mention that 1 export is necessary, but it should be done via BetterBibTex as via this way, the automatic exporting can be set up

Reply:
yes, perfect, that's what I meant!